notebooks/tutorials/additional_tutorials/Multilabel_ResNet_on_HSN.ipynb

```diff
@@ -0,0 +1,1155 @@
+    "\n",
+    "> In this step and the next one, there are some additional codes which show how you can visualize your sample\n",
+    "\n",
+    "> First load the test dataset by addressing it (\"DBD-research-group/BirdSet\") and specifying the name of dataset (\"HSN\") and selecting the split between (train/test/val).\n",
+    "In this part by setting offset and duration you can just take a bird sound from an audio sample.\n",
+    "\n",
+    "> If you want to listen to the audio, see the waveform or the spectrogram of the audio, you need to follow steps in the next section.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "efa9f9f63c1a47429c8afbf3d80a29a9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating train split: 0 examples [00:00, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "487b781a337748bfbe5d2c1c42ed7fc1",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating test split: 0 examples [00:00, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ecfda4dbf81c478cbb94b4030c8b2f87",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating test_5s split: 0 examples [00:00, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "## load the test dataset\n",
+    "from datasets import load_dataset\n",
+    "hsn_test = load_dataset(\"DBD-research-group/BirdSet\",\"HSN\", split=\"test\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get an audio sample\n",
+    "import librosa\n",
+    "import torchaudio\n",
+    "sample = 199\n",
+    "sr = 32000\n",
+    " \n",
+    "sample_audio =  librosa.load(\n",
+    "    hsn_test[sample]['filepath'],\n",
+    "    sr=sr,\n",
+    "    offset=hsn_test[sample]['start_time'],\n",
+    "    duration=hsn_test[sample]['end_time'] - hsn_test[sample]['start_time'])\n",
+    "sample_tensor = torchaudio.load(\n",
+    "        hsn_test[sample]['filepath'],\n",
+    "        normalize=True,\n",
+    "        frame_offset=hsn_test[sample]['start_time'] * sr,\n",
+    "        num_frames=(hsn_test[sample]['end_time'] - hsn_test[sample]['start_time']) * sr\n",
+    "        )\n",
+    "sample_audio = sample_audio[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 6: Visualization"
```

Comment:
@PariaValizadeh Could you move the visualization before the model training? So that the reader sees how the data gets preprocessed by the birdset pipeline?